routes/dao_coin_exchange.go

```diff
@@ -0,0 +1,183 @@
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/deso-protocol/core/lib"
+	"github.com/holiman/uint256"
+	"io"
+	"math/big"
+	"net/http"
+)
+
+type GetDAOCoinLimitOrdersRequest struct {
+	DAOCoin1CreatorPublicKeyBase58CheckOrUsername string `safeForLogging:"true"`
+	DAOCoin2CreatorPublicKeyBase58CheckOrUsername string `safeForLogging:"true"`
+}
+
+type GetDAOCoinLimitOrdersResponse struct {
+	Orders []DAOCoinLimitOrder
+}
+
+type DAOCoinLimitOrder struct {
```

Comment:
We typically use the `EntryResponse` suffix to denote a differentiation between the core struct and the backend struct.